src/exchanges/huobi-client.js

```diff
@@ -0,0 +1,66 @@
+const BasicClient = require("../basic-client");
+const Trade = require("../trade");
+const zlib = require('zlib');
+
+class HuobiClient extends BasicClient {
+  constructor() {
+    super("wss://api.huobi.pro/ws", "Huobi");
+  }
+
+  _sendSubscribe(remote_id) {
+    this._wss.send(
+      JSON.stringify({
+        sub: `market.${remote_id}.trade.detail`,
+        id: ''
+      })
+    );
+  }
+
+  _sendUnsubscribe(remote_id) {
+    this._wss.send(
+      JSON.stringify({
+        unsub: `market.${remote_id}.trade.detail`,
+        id: ''
+      })
+    );
+  }
+
+  _onMessage(raw) {
+    new Promise((resolve, reject) => {
```

Comment:
Hmm... we're going to want to improve this method just because it's going to get called ALOT.  In thinking about it, I think we need to remove the promise.  We should also log errors as I don't think there is an error emission, so lets just log it for now using winston.error.